src/main/resources/default/templates/system/api.html.pasta

```diff
@@ -92,7 +92,7 @@
                                     <td>
                                         <i:raw>@expandMessage(smartTranslate(parameter.description()))</i:raw>
                                     </td>
-                                    <td>@toUserString(parameter.required())</td>
+                                    <td>@parameter.required()</td>
```

Comment:
To improve the rendering a little bit we could maybe choose two unicode chars for true/false
Something like ✓ and ✗ for example
Or we could use a read-only checkbox element

Reply:
Feels to me like not much benefit, but if there are more votes for this, I can add it